app/screens/network/Network.js

```diff
@@ -0,0 +1,207 @@
+// @flow
+import React, { Component } from 'react';
+import { RouterHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import { connect } from 'react-redux';
+import { ScreenErrorBoundary } from '/components/errorHandler';
+import { WrapperWith2SideBars, Link, NetworkIndicator, Tooltip, ProgressBar } from '/basicComponents';
+import { smColors } from '/vars';
+import { network } from '/assets/images';
+import { getNodeStatusText, getFormattedTimestamp } from '/infra/utils';
+import { eventsService } from '/infra/eventsService';
+
+const isDarkModeOn = localStorage.getItem('dmMode') === 'true';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  justify-content: space-between;
+`;
+
+const DetailsWrap = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const FooterWrap = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+const Message = styled.div`
+  display: flex;
+  flex-direction: row;
+  color: ${({ color }) => color};
+`;
+
+const DetailsRow = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: ${({ isLast }) => (isLast ? `0px` : `1px solid ${isDarkModeOn ? smColors.white : smColors.darkGray10Alpha};`)};
+`;
+
+const DetailsText = styled.div`
+  font-size: 16px;
+  line-height: 20px;
+  margin: 10px 0;
+  color: ${isDarkModeOn ? smColors.white : smColors.realBlack};
+`;
+
+const GrayText = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  font-size: 14px;
+  text-transform: uppercase;
+  color: ${smColors.dark75Alpha};
+`;
+
+const DetailsTextWrap = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`;
+
+const Progress = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100px;
+  margin-left: 20px;
+`;
+
+const ProgressLabel = styled.div`
+  margin-left: 10px;
+`;
+
+type Props = {
+  node: Object,
+  status: Object,
+  nodeIndicator: Object,
+  history: RouterHistory
+};
+
+class Network extends Component<Props, State> {
+  startUpDelay = 5; // eslint-disable-line react/sort-comp
+
+  noPeersCounter = 0; // eslint-disable-line react/sort-comp
+
+  color;
+
+  render() {
+    const { node, status, nodeIndicator } = this.props;
+
+    const networkName = 'TweedleDee 0.1.0';
+
+    return (
+      <WrapperWith2SideBars width={1000} height={500} header="NETWORK" headerIcon={network} subHeader={networkName}>
+        <Container>
+          <Message color={nodeIndicator.color}>{nodeIndicator.message}</Message>
+          <DetailsWrap>
+            <DetailsRow>
+              <DetailsTextWrap>
+                <DetailsText>Age</DetailsText>
+                <Tooltip width="250" text="tooltip age" />
+              </DetailsTextWrap>
+              <GrayText>{getFormattedTimestamp(node.genesisTime)}</GrayText>
```

Comment:
connect 'genesisTime' instead of whole 'node' object

Reply:
fix